site/content/in-dev/unreleased/metastores.md

```diff
@@ -24,12 +24,11 @@ weight: 700
 
 This page documents important configurations for connecting to a production database through [EclipseLink](https://eclipse.dev/eclipselink/).
 
-## Building Polaris with EclipseLink
+## Polaris EclipseLink
 
-Polaris doesn't ship with any JDBC driver. You must specify them when building Polaris with
-EclipseLink by using Gradle's project property:
-`-PeclipseLinkDeps=<jdbc-driver-artifact1>,<jdbc-driver-artifact2>,...`. See below examples for H2
-and Postgres.
+Polaris includes EclipseLink plugin by default with PostgreSQL driver.
+
+In order to add other JDBC drivers, you have to build Polaris using the `eclipseLinkDeps` build property.
```

Comment:
I think we can remove line 136 to 139 as well, since Postgres doesn't need a special build any more
```
./gradlew \
  :polaris-quarkus-server:assemble \
  :polaris-quarkus-server:quarkusAppPartsBuild --rerun \
  -PeclipseLinkDeps=org.postgresql:postgresql:42.7.4
```

Reply:
Good catch ! Let me clean this section.